### PR TITLE
BGDIINF_SB-2569: Makefile.ireland and README_IRELAND.md are obsolete and can be deleted

### PR DESCRIPTION
**Pull request by hansmannj**

### CODE REVIEW

**Comment by hansmannj** [comment]
CodeBuild was successful, just did not report this back.
Just waiting for your approval @ltshb

**Review by ltshb — APPROVED** [review]

**Pull request merged**
